client/my-sites/hosting/sftp-card/index.js

```diff
@@ -239,8 +241,21 @@ export const SftpCard = ( {
 				"Access and edit your website's files directly by creating SFTP credentials and using an SFTP client."
 		  );
 
+	const [ isReauthRequired, setIsReauthRequired ] = useState(
+		twoStepAuthorization.isReauthRequired()
+	);
+
+	const onTwoStepChange = ( { reauthStatus } ) => {
+		setIsReauthRequired( reauthStatus );
```

Comment:
We can merge the PR once we fix this, @vykes-mac:

The param is `isReauthRequired`, not `reauthStatus`, so we must update this handler accordingly:

```suggestion
	const onTwoStepChange = ( { isReauthRequired } ) => {
		setIsReauthRequired( isReauthRequired );
```